rasa/core/domain.py

```diff
@@ -118,9 +139,31 @@ def from_dict(cls, data: Dict) -> "Domain":
             utter_templates,
             data.get("actions", []),
             data.get("forms", []),
+            session_config=session_config,
             **additional_arguments,
         )
 
+    @staticmethod
+    def _get_session_config(additional_arguments: Dict) -> SessionConfig:
+
+        session_length = additional_arguments.pop(SESSION_LENGTH_KEY, None)
+
+        # TODO: 2.0 reconsider how to apply sessions to old projects and legacy trackers
+        if session_length is None:
+            warnings.warn(
+                "No tracker session configuration was found in the loaded domain. "
+                "Domains without a session config will be deprecated in Rasa "
+                "version 2.0.",
+                DeprecationWarning,
```

Comment:
Should be a `FutureWarning` (see code conventions)

Reply:
👍